lib/Zonemaster/Backend/TestAgent.pm

```diff
@@ -214,11 +214,12 @@ sub to_idn {
     my ( $self, $str ) = @_;
 
     if ( $str =~ m/^[[:ascii:]]+$/ ) {
+        $str = lc $str;
```

Comment:
I think this should be done in RPCAPI.pm.

Reply:
The ASCII labels is only available here. In RPCAPI, you could have IDN.

Reply:
What I mean is that if the domain is put in lower case by the RPCAPI, then TestAgent.pm will never see anything else.